### DISCUSSION TITLE
Should I delete this thread?

### DISCUSSION THREAD

**Discussion by maybeJosiah** [community/community#142333] [New to GitHub] [answered]
[Thread](https://github.com/orgs/community/discussions/142160), I know at least 2 people have reported this spam, I feel like I promoted too much but I don't know how else to ask for code review and algorithm review than to say here it is, I know one report was because "maybe" stuff, how exactly is that post spammy and should I delete it?

**Comment by buritbest123** [comment, marked as answer]
It sounds like you're concerned about your post being perceived as spam. While promoting your code for review is important, it might be seen as spam if it's overly frequent, lacks context, or seems unrelated to the ongoing discussions.

Here are a few things to consider to avoid your post being marked as spam:
- **Frequency:** If you're sharing the same message or request multiple times, it might come across as spamming.
- **Context:** Make sure your post aligns with the conversation and adds value. If it feels disconnected from the thread, it could be reported.
- **Clarity:** Make sure your request is clear and focused. If there's uncertainty like "maybe," it could seem less relevant or genuine.

To avoid issues, try to engage with the community first by contributing feedback on others' work before asking for reviews. That way, your posts feel more collaborative and less self-promotional.

As for deleting the post, if you think it adds value and wasn't intended to spam, you could clarify your intentions in a follow-up comment. However, if it's being flagged consistently, it might be a good idea to reassess and delete it to avoid further reports.

**Reply by maybeJosiah** [reply]
I am trying to promote as little as possible but I am trying for code review, impossible to get without saying what things are maybe or maybe not, maybe.

**Comment by ghostinhershell** [comment]
Hey @maybeJosiah, welcome to our community 👋🏾 

Other than echoing @buritbest123 advice here that sharing the same message or request multiple times might come across as spamming, I think your original post is fine and doesn't need to be deleted. I hope you're able to get the feedback that you're looking for and look forward to seeing where your development journey takes you! 🚀 

Cheers!
